ygnmi/gnmi.go

```diff
@@ -73,7 +73,7 @@ func subscribe[T any](ctx context.Context, c *Client, q AnyQuery[T], mode gpb.Su
 			dt = gpb.GetRequest_STATE
 		}
 		sub = &getSubscriber{
-			client:   c.gnmiC,
+			client:   c,
```

Comment:
A little torn between whether it's worth storing a reference to the full client here or whether the gnmi client and logging level should be two fields. Daniel can make the call :)

Reply:
i don't have a strong opinion on this

Reply:
I'll keep it as is then. This is an unexported field and we can modify this later.